Source/WebCore/page/Page.h

```diff
@@ -280,6 +280,7 @@ enum class RenderingUpdateStep : uint32_t {
 #endif
     RestoreScrollPositionAndViewState = 1 << 24,
     AdjustVisibility                  = 1 << 25,
+    SnapshottedScrollOffsets          = 1 << 26,
```

Comment:
I think it would be nice to keep these in the order in which `Page::updateRendering()` consults them.

Reply:
Existing ones are not in order either. I moved the offset update last though.